docs/dev-guide/web-sdk.md

```diff
@@ -0,0 +1,74 @@
+---
+id: dev-guide-web-sdk
+title: Web SDK
+---
+
+The Jitsi Meet Web SDK provides the same user experience as the Jitsi Meet app, in a customizable way which you can embed in your apps.
+
+## Sample applications using the SDK
+If you want to see how easy integrating the Jitsi Meet Web SDK into a web application is, take a look at the [sample applications directory](https://github.com/jitsi/jitsi-meet-web-sdk/tree/main/examples).
+
+## Installation
+To access the Web SDK modules in your application you need to install it as a dependency:
+```bash
+npm install @jitsi/web-sdk
+```
+
+## Modules
+
+### fetchExternalApi
```

Comment:
Please document the components first, this is a lower level APi which we expect less people to use.